test/test_sampler_pbs.py

```diff
@@ -0,0 +1,107 @@
+import os
```

Comment:
Introduce a [pytest marker](http://doc.pytest.org/en/latest/example/markers.html) 'pbs' that depends on the PBS system. Make a script in the test directory to run all pbs tests. Skip pbs tests on CircleCI.